pkg/tsdb/testdatasource/stream_handler.go

```diff
@@ -55,31 +58,31 @@ func (s *Service) PublishStream(_ context.Context, req *backend.PublishStreamReq
 func (s *Service) RunStream(ctx context.Context, request *backend.RunStreamRequest, sender *backend.StreamSender) error {
 	s.logger.Debug("New stream call", "path", request.Path)
 	var conf testStreamConfig
-	switch request.Path {
-	case "random-2s-stream":
+	switch {
+	case request.Path == "random-2s-stream":
 		conf = testStreamConfig{
 			Interval: 2 * time.Second,
 		}
-	case "random-flakey-stream":
+	case request.Path == "random-flakey-stream":
 		conf = testStreamConfig{
 			Interval: 100 * time.Millisecond,
 			Drop:     0.75, // keep 25%
 		}
-	case "random-labeled-stream":
+	case request.Path == "random-labeled-stream":
 		conf = testStreamConfig{
 			Interval: 200 * time.Millisecond,
 			Drop:     0.2, // keep 80%
 			Labeled:  true,
 		}
-	case "random-20Hz-stream":
-		conf = testStreamConfig{
-			Interval: 50 * time.Millisecond,
-		}
-	case "flight-5hz-stream":
+	case request.Path == "flight-5hz-stream":
 		conf = testStreamConfig{
 			Interval: 200 * time.Millisecond,
 			Flight:   newFlightConfig(),
 		}
+	case random20HzStreamRegex.MatchString(request.Path):
```

Comment:
[strings.HasPrefix](https://pkg.go.dev/strings#HasPrefix) may be easier here

Reply:
@ArturWierzbicki -- can you split out the backend streaming changes into its own PR?  I don't think we need any of the complications of environment variables,  This datasoure only exists in "test" environments anyway

Reply:
extracted https://github.com/grafana/grafana/pull/43295